client/src/entry/analysis/menu.js

```diff
@@ -105,6 +105,7 @@ export function fetchMenu(options = {}) {
                 {
                     title: _l("Datasets"),
                     url: "/datasets/list_published",
```

Comment:
```suggestion
                    url: "/datasets/list",
```

Reply:
there is no grid for public datasets, this url was added erroneously during the transition under "Data" in masthead